lib/deprecation_toolkit/minitest_hook.rb

```diff
@@ -1,17 +1,12 @@
 # frozen_string_literal: true
 
 require "minitest"
+require 'deprecation_toolkit/test_triggerer'
```

Comment:
Can we autoload this file instead of requiring it

Reply:
Done!